src/services/config.service.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2018 Amsterdam University of Applied Sciences (AUAS)
+ *
+ * This software is distributed under the terms of the
+ * GNU General Public Licence version 3 (GPL) version 3,
+ * copied verbatim in the file "LICENSE"
+ */
+
+import * as dotenv from 'dotenv';
+import * as fs from 'fs';
+
```

Comment:
Please provide a comment what the goal is of this class.

Reply:
With the ConfigService class you can read specific values from an .env file in any class with the following syntax `configService.get('Key')`

Reply:
Added comment